openedx/core/djangoapps/course_groups/tests/test_cohorts.py

```diff
@@ -253,8 +253,8 @@ def test_get_cohort(self):
         )
 
     @ddt.data(
-        (True, 2),
-        (False, 6),
+        (True, 8), # 6 Django savepoints & 2 cohort SELECTs
+        (False, 12), # 6 Django savepoints & 6 cohort SELECTs
```

Comment:
I hope there's a way to suppress some of that: 2 real selects, and 6 savepoints!?